library/src/main/java/com/applandeo/materialcalendarview/CalendarUtils.kt

```diff
@@ -0,0 +1,73 @@
+@file:JvmName("CalendarUtils")
+
+package com.applandeo.materialcalendarview
+
+import android.content.Context
+import android.graphics.*
+import android.graphics.drawable.BitmapDrawable
+import android.graphics.drawable.Drawable
+import androidx.core.content.ContextCompat
+import java.util.*
+import java.util.concurrent.TimeUnit
+
+/**
+ * Created by Mateusz Kornakiewicz on 03.08.2018.
```

Comment:
Ja bym chciał to usunąć z każdego pliku 😉 bo to już nie jest mój kod tylko kod przekonwertowany.